Feature/kas 2760 pub translations  sort

mdv-professional opened the pull request:
[Jira](https://kanselarij.atlassian.net/browse/KAS-2760)

gebruikte project-branch: feature/KAS-2756-pub-proof--decisions

**Notes**
- Is het wel goed [om de modal state niet te resetten](https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/1006/commits/58c73a6f261b549077eb05c6086eba006fd93e4e)? Nu kunt ge, al is het gezocht, back en forward gaan in uw browser en uw modal parameters zijn veranderd.

Commit:
KAS-2750 feature                                  | sort [pub translation]

Commit:
KAS-2750 fix refresh                              | sort [pub translation]

Commit:
KAS-2760 no nowrap in prototype                   | sort [pub translation]

erikap commented on app/pods/publications/publication/translations/documents/controller.js:
```diff
@@ -28,6 +64,12 @@ export default class PublicationsPublicationTranslationsDocumentController exten
 
+  @action
+  changeSorting(sortingString) {
+    this.sortingString = sortingString;
+    this.set('qpSortingString', sortingString);
```
Waarom heb je beiden nog nodig? Is enkel `qpSortingString` niet voldoende?

mdv-professional replied:
qp's worden liever niet getracked omdat die performantieissues zou geven.

erikap replied:
Dat klopt, maar dan heb je voldoende aan de property die geupdate wordt met `set()`. Om te zorgen dat de `get pieces()` geupdate worden wanneer de sortering wijzigt, moet je deze annoteren met `@computed('sort')`. Dit is nog afkomstig uit het pre-Ember Octane tijdperk. Eens we geupgrade zijn en er een `@tracked sort` is, kan de `@computed` annotatie weg.

Ik heb de change gecommit met de nodige TODOs voor later zodanig dat deze PR kan doorstromen naar development.

erikap committed:
Remove duplicate sorting var in translations/publication documents

Until we upgrade to Ember v3.22+ making query params `@tracked` has a
performance penalty. Therefore we use a regular property which gets
updated using `set()` and trigger updating of a `get`-function depending
on the property using `@computed`

erikap committed:
Remove unused eslint-disable flag
